deeds-dapp-service/src/main/java/io/meeds/deeds/service/MeedTokenMetricService.java

```diff
@@ -56,6 +66,9 @@ public class MeedTokenMetricService {
   @Getter
   private List<String>               lockedPolygonAddresses;
 
+  @Value("${meeds.exchange.Coingecko.UsdPrice:https://api.coingecko.com/api/v3/simple/price?ids=meeds-dao&vs_currencies=USD}")
+  private String InstantMeedUsdPrice;
```

Comment:
Formatting issue and the variable name should start with a lower case, in addition, the name isn't significant IMO 
```suggestion
  private String meedUsdPriceProviderUrl;
```